workflow/workflow.go

```diff
@@ -217,6 +217,16 @@ func GetMetricsScope(ctx Context) tally.Scope {
 	return internal.GetMetricsScope(ctx)
 }
 
+// GetTotalHistoryBytes returns the current history size of that workflow
+func GetTotalHistoryBytes(ctx Context) int64 {
+	return internal.GetTotalEstimatedHistoryBytes(ctx)
+}
+
+// GetHistoryCount returns the current number of history event of that workflow
+func GetHistoryCount(ctx Context) int64 {
```

Comment:
where is `GetHistoryCount` used? If not used or planing to use in the future let's remove.

Reply:
Also form the 2 other places this function is defined

Reply:
They will be used as API calls from client code

Reply:
@timl3136  did Steven's concenrs about raising something which might be nondeterministic at runtime get resolved? I don't have enough context to be useful here, but please make sure that issue's resolved before landing.